src/synth.ts

```diff
@@ -407,6 +425,18 @@ function initializeAperiodic(audioContext: BaseAudioContext) {
 
     return new AperiodicWave(audioContext, freqs, amps, maxNumberOfVoices, tolerance)
   })
+
+  getInharmonicSimpleWaveformNames().forEach((id) => {
+    APERIODIC_WAVES[id] = computed(() => {
+      const freqs = getInharmonicSimple(id)
+      const nsm1 = [...freqs.keys()]
```

Comment:
I can't figure out what this is short for. Maybe `indices` instead?

Reply:
that was “`ns` where `n` is really `n` minus 1”, yeah, probably better off without